test/inductor/test_compile_subprocess.py

```diff
@@ -155,6 +165,7 @@ def baseline(x, y):
         self.assertGreater(
             do_bench(lambda: baseline(x, y)), do_bench(lambda: optimized(x, y))
         )
+        self.assertTrue("'max_autotune': True" in source_codes[-1])
```

Comment:
I guess it would be a race condition to expect there to always be two...
Consider something like:
```
self.assertTrue(len(source_codes) == 1 or "'max_autotune': True" not in source_codes[0])
self.assertTrue("'max_autotune': True" in source_codes[-1])
```

Reply:
It actually isn't a race condition since we barrier on the first compile finishing before checking the futures. It would be for compiles 2 onwards though... 

```
        if self.post_compile_data:
            for i, future in enumerate(self.progression_futures):
                if future.done():
                    stage_index = i
```